youtube_dl/extractor/npr.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import os.path
+import re
+
+from ..compat import compat_urllib_parse_unquote
+from ..utils import url_basename
+from .common import InfoExtractor
+
+class NprIE(InfoExtractor):
+    _VALID_URL = r'http://(?:www\.)?npr\.org/player/v2/mediaPlayer.html?.*id=(?P<id>[0-9]+)'
+    _TEST = {
+    'url': 'http://www.npr.org/player/v2/mediaPlayer.html?id=445367719',
+    'info_dict': {
+        'id': '445367719',
+        'ext': 'mp4',
+        'title': 'VEGA INTL. Night School'
+    }
+}
+
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        video_id = mobj.group('id')
+        webpage_url = 'http://www.npr.org/player/v2/mediaPlayer.html?id=' + video_id
+        webpage = self._download_webpage(webpage_url, video_id)
+        key = 'MDAzMzQ2MjAyMDEyMzk4MTU1MDg3ZmM3MQ010'
+        xml_url = 'http://api.npr.org/query?id=%s&apiKey=%s' % (video_id, key)
+
+        config = self._download_xml(xml_url,video_id, note='Downloading XML')
+
+        audio = config.findall('./list/story/audio[@type="standard"]')
```

Comment:
This syntax is not supported in python 2.6. You should use `xpath_*` from `utils`.